Tiltfile

```diff
@@ -87,6 +87,18 @@ if os.getenv('INITIUM_DEPLOY_MINIMAL', 'false') == 'false':
         namespace='argocd',
     ))
 
+    ## Kubernetes replicator
+    k8s_yaml(namespace_yaml('kubernetes-replicator'), allow_duplicates=False)
+    local('helm dependency update ./addons/kubernetes-replicator')
+    local('helm repo update')
+
+    k8s_yaml(helm(
+        './addons/kubernetes-replicator',
+        name='kubernetes-replicator',
+        namespace='kubernetes-replicator',
+    ))
```

Comment:
@jeevanions this should not be here, the replicator is installed as part of the addons using the app of apps

Reply:
the addon is missing from the `app-of-apps/values.yaml` file, you should add it there the get it installed